load.Rmd

```diff
@@ -378,7 +382,6 @@ of 5.0%.
 ## Scenarios
 
 ```{r, include=FALSE}
-l$slc <- mod_logsqr$fitted.values
 l$base_olc <- l$Bicycle
 l$base_slc <- l$slc * l$All
 l$base_sic <- l$base_slc - l$base_olc
```

Comment:
Agree, the duplication here is not needed - suggest removing the new line 385. `base_slc` is different from govtarget - I think that name's OK.